pkg/asset/internal/templates.go

```diff
@@ -430,6 +437,43 @@ spec:
     type: RollingUpdate
 `)
 
+var CheckpointerServiceAccount = []byte(`apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: kube-system
+  name: pod-checkpointer
+`)
+
+// TODO: Drop checkpointer RBAC resources to a Role and RoleBinding if
+// the checkpoint switches to only watching kube-system.
```

Comment:
Should we just make this change? It only needs to watch kube-system (also can be follow-up PR)

Reply:
I can open a separate PR. We'd have to wait the checkpointer change to merged before we drop this to a role, so we can't do it in one PR anyway.

Reply:
Why would the checkpointer have to be changed? It just wouldn't have access to non-kube-system right?

Reply:
Checkpointer attempts to watch all namespaces. If we restrict it now that request will fail.

https://github.com/kubernetes-incubator/bootkube/blob/v0.8.2/pkg/checkpoint/apiserver.go#L18

Reply:
Opened the checkpointer PR https://github.com/kubernetes-incubator/bootkube/pull/774